src/v2/mappers/ScheduleMapper.ts

```diff
@@ -57,6 +57,18 @@ export class ScheduleMapper {
     };
   }
 
+  getSimpleEvent (event: DbEvent): SimpleEventResponse {
+    return {
+      id: event.id,
+      name: event.name,
+      startTime: event.startTime,
+      endTime: event.endTime,
+      period: event.period,
+      url: event.url || null,
+      eventInfo: event.eventInfo[0]?.description || null,
```

Comment:
now get endpoints return events without eventInfo, so you need improve 3 endpoints for get day/week/fortnight events and add field eventInfo

I think you can just add this field in getTelegramEvents